src/components/webview/IntuitaPanelProvider.ts

```diff
@@ -378,19 +369,19 @@ export class IntuitaPanelProvider {
 						const searchParams = new URLSearchParams();
 
 						searchParams.set(
-							CS_SEARCH_PARAMS_KEYS.BEFORE_SNIPPET,
+							SEARCH_PARAMS_KEYS.BEFORE_SNIPPET,
 							encode(beforeSnippet),
 						);
 						searchParams.set(
-							CS_SEARCH_PARAMS_KEYS.AFTER_SNIPPET,
+							SEARCH_PARAMS_KEYS.AFTER_SNIPPET,
 							encode(afterSnippet),
 						);
 						searchParams.set(
-							CS_SEARCH_PARAMS_KEYS.CODEMOD_NAME,
+							SEARCH_PARAMS_KEYS.CODEMOD_NAME,
 							encode(job.codemodName),
 						);
 						// searchParams.set(
-						// 	CS_SEARCH_PARAMS_KEYS.CODEMOD_SOURCE,
+						// 	SEARCH_PARAMS_KEYS.CODEMOD_SOURCE,
```

Comment:
perhaps we can remove the commented-out code?

Reply:
addressed